File: InsertionSort/InsertionSort2.c
#include<stdio.h>

int main()
{
int x[5];
int y,e,f,num,m;
m=5-1;
for(y=0;y<=m;y++)
{
printf("Enter a number : ");
scanf("%d",&x[y]);
fflush(stdin);
}
for(e=1;e<=m;e++)
{
for(num=x[e],f=e-1;f>=0 && x[f]>num;f--) x[f+1]=x[f];
x[f+1]=num;
}
for(y=0;y<=m;y++) printf("%d\n",x[y]);
return 0;
}

File: InsertionSort/InsertionSort2.py
def insertionSort(x):
    m=len(x)-1
    e=1
    while e<=m:
        num=x[e]
        f=e-1
        while f>=0 and x[f]>num:
            x[f+1]=x[f]
            f-=1
        x[f+1]=num
        e+=1
x=[]
req=5
m=req-1
for i in range(req): x.append(int(input("Enter a number : ")))
insertionSort(x)
for i in x: print(i)
